packages/ui/src/components/DirectionOfEffectIcon.tsx

```diff
@@ -0,0 +1,88 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { naLabel } from "../constants";
+import {
+  faArrowTrendDown,
+  faArrowTrendUp,
+  faCircleExclamation,
+  faMinus,
+  faQuestion,
+  faShieldHalved,
+} from "@fortawesome/free-solid-svg-icons";
+import { makeStyles } from "@mui/styles";
+import { Box, Divider, Tooltip } from "@mui/material";
+
+const useStyles = makeStyles(theme => ({
+  colorBlue: {
+    color: theme.palette.primary.main,
+  },
+  tooltip: {
+    backgroundColor: `${theme.palette.background.paper} !important`,
+    border: `1px solid ${theme.palette.grey[300]}`,
+    color: `${theme.palette.text.primary} !important`,
+  },
+}));
+
+type DirectionOfEffectIconProp = {
+  variantEffect?: string;
+  directionOnTrait?: string;
+};
+
+const LoF = "LoF";
+const RISK = "risk";
+
+function DirectionOfEffectIcon({ variantEffect, directionOnTrait }: DirectionOfEffectIconProp) {
+  const classes = useStyles();
+
+  let tooltipText = "";
+  if (variantEffect === LoF) tooltipText += "Loss of Function │ ";
+  else if (variantEffect) tooltipText += "Gain of Function │ ";
+  else tooltipText += naLabel + " │ ";
+
+  if (directionOnTrait === RISK) tooltipText += "Risk";
+  else if (directionOnTrait) tooltipText += "Protective";
+  else tooltipText += naLabel;
```

Comment:
Here you can be more _semantic_ making this block a function. ej `const tooltipText = getTooltipText(...);`